HW02_B_Mimic.md

```diff
@@ -37,6 +37,17 @@ data("diamonds")
 Using the diamonds dataset, make this graph:
 ![](HW02_B_Mimic_files/figure-gfm/unnamed-chunk-2-1.png)<!-- -->
 
+```{r graph1 code, echo=FALSE}
+ggplot(diamonds, aes(x = cut, fill = clarity)) + 
+  geom_bar(position = "dodge") +
+  xlab("Diamond Cut") +
+  ylab("Number of Diamonds") +
+  labs(title = "My Diamond Collection", subtitle = "Boxplot representing the number of diamonds in my diamond collection by \ntype of cut quality and clarity of diamond") +

```

Comment:
I have a subtle suggestion, that is removing the blank before "\n", I'm not sure if it will but I'm afraid it may leave a blank before starting a new line, and may affect the layouts of the text (like, not actually centered).

Reply:
Lai, thanks for the suggestion! I went back to RStudio to double check if that was the case. Apparently, doing it my way or as per your suggestion both give the same result - it didn't leave a blank/space before starting a new line as you can see from my graph. However, if I had a blank between \n and the word that came after it (in this case \ntype vs \n type), this will give us a blank before starting a new line. This is a good observation nevertheless, it is always in the little things that can make or break our graphs! :)